src/JJMasterData.Commons/Language/ILocalizationProvider.cs

```diff
@@ -2,17 +2,17 @@
 
 namespace JJMasterData.Commons.Language;
 
-public interface ITranslator
+public interface ILocalizationProvider
```

Comment:
I renamed this because ITranslator **do not** make translations, it just return **localized values**.